### PR TITLE
Olga/931 locality changes places

### PR DESCRIPTION
**Pull request by okavvada**
Connect to #931 and #934.

- [x] Update tests
- [ ] Update data migrations
- [x] Update docs

Ready for review! @rmarianski , @nvkelso , @zerebubuth

### CODE REVIEW

File: docs/layers.md [hunk 1]
```diff
@@ -399,7 +399,7 @@ Places with `kind` values of `continent`, `country`, with others added starting
 #### Place properties (common optional):
 
 * `capital`: a `true` value normalizes values between OpenStreetMap and Natural Earth for kinds of `Admin-0 capital`, `Admin-0 capital alt`, and `Admin-0 region capital`.
-* `state_capital`: a `true` value normalizes values between OpenStreetMap and Natural Earth for kinds of `Admin-1 capital` and `Admin-1 region capital`.
+* `region_capital`: a `true` value normalizes values between OpenStreetMap and Natural Earth for kinds of `Admin-1 capital` and `Admin-1 region capital`.
```
Comment (rmarianski) [inline docs/layers.md]: Do we also need to update the place kind values below? Seems like some of those are no longer possible, and normalized versions of them are in `kind_detail`.

File: docs/layers.md [hunk 2]
```diff
@@ -391,45 +391,39 @@ Places with `kind` values of `continent`, `country`, with others added starting
...
 * `neighbourhood`
-* `Populated place`
 * `province`
-* `quarter`
-* `Scientific station`
+* `scientific_station`
 * `state`
-* `suburb`
 * `town`
 * `village`
```
Comment (okavvada) [inline docs/layers.md]: @rmarianski , I updated the kind and kind_detail values with the ones that currently exist in the `places.yaml`. @nvkelso , can you check to make sure you agree with the changes?

File: docs/layers.md [hunk 3]
```diff
@@ -391,45 +391,39 @@ Places with `kind` values of `continent`, `country`, with others added starting
 
 * `name`
 * `id`: osm_id from OpenStreetMap or Natural Earth id
-* `kind`: the original value of the OSM `place` tag and Natural Earth `featurecla`
+* `kind`: normalizes values between OpenStreetMap and Natural Earth
```
Comment (nvkelso) [inline docs/layers.md]: `normalized`

File: docs/layers.md [hunk 4]
```diff
@@ -391,45 +391,39 @@ Places with `kind` values of `continent`, `country`, with others added starting
...
 #### Place properties (common optional):
 
 * `capital`: a `true` value normalizes values between OpenStreetMap and Natural Earth for kinds of `Admin-0 capital`, `Admin-0 capital alt`, and `Admin-0 region capital`.
-* `state_capital`: a `true` value normalizes values between OpenStreetMap and Natural Earth for kinds of `Admin-1 capital` and `Admin-1 region capital`.
+* `region_capital`: a `true` value normalizes values between OpenStreetMap and Natural Earth for kinds of `Admin-1 capital` and `Admin-1 region capital`.
 * `labelrank`: labelrank value from Natural Earth
 * `min_zoom`: Currently neighbourhoods only, from Who's On First
 * `max_zoom`: Currently neighbourhoods only, from Who's On First
 * `is_landuse_aoi`: Currently neighbourhoods only, from Who's On First
+* `kind_detail`: the original value of the OSM `place` tag and Natural Earth `featurecla`, see below.
```
Comment (nvkelso) [inline docs/layers.md]: Per @rmarianski's note in https://github.com/tilezen/vector-datasource/pull/942/files#r73375861, are we normalizing them or passing thru thru? (I think we're both white listing and perhaps normalizing some?)

File: docs/layers.md (hunk 1 above)
Comment (nvkelso) [inline docs/layers.md]: Looks like this was addressed in https://github.com/tilezen/vector-datasource/pull/942/commits/8b415af33acfb320baba7775696905ed7a0b7baa and https://github.com/tilezen/vector-datasource/pull/942/commits/1b577aee809e04564dd130788fcff5c7842819e9.

File: docs/layers.md (hunk 2 above)
Comment (nvkelso) [inline docs/layers.md]: LGTM

File: docs/layers.md (hunk 4 above)
Comment (nvkelso) [inline docs/layers.md]: I think this language is okay. We have some other cleanup to do in this document, but right now this language matches the existing writing style.

File: docs/layers.md [hunk 5]
```diff
@@ -391,45 +391,39 @@ Places with `kind` values of `continent`, `country`, with others added starting
...
 
 #### Place kind values:
 
-* `Admin-0 capital alt`
-* `Admin-0 capital`
-* `Admin-0 region capital`
-* `Admin-1 capital`
-* `Admin-1 region capital`
-* `borough`
```
Comment (nvkelso) [inline docs/layers.md]: We actually still support this, but it's not obvious. See: https://github.com/tilezen/vector-datasource/blob/1b577aee809e04564dd130788fcff5c7842819e9/queries/places.jinja2#L60. 

The full list of WOF values is [here](https://github.com/tilezen/vector-datasource/blob/d7059dcac4954f29e4ff862aaec8468d795b970f/data/schema.sql#L8-L12), please update the `kind` list in the docs to:

- add back `borough`
- add `microhood`, and
- add `macrohood`

**Comment by nvkelso** [comment]
Once https://github.com/tilezen/vector-datasource/pull/942#discussion_r73442227 and https://github.com/tilezen/vector-datasource/pull/942#discussion_r73441033 are fixed, ready to merge.

File: docs/layers.md (hunk 4 above)
Comment (okavvada) [inline docs/layers.md]: @nvkelso what do you mean by passing thru thru?

Comment (nvkelso) [inline docs/layers.md]: In the case of WOF, we're passing thru kind values (without examining them), where with OSM we're examining each and either say's it's in a source whitelist and assigning a tilezen output whitelist value that is the "same" (passing thru, but whitelisted) or "different" (transforming).

Comment (nvkelso) [inline docs/layers.md]: (Nothing more to do here.)

File: docs/layers.md [hunk 6]
```diff
@@ -412,6 +412,9 @@ Places with `kind` values of `continent`, `country`, with others added starting
 * `region`
 * `locality`
 * `neighbourhood`
+* `microhood`
+* `macrohood`
+* `borough`
```
Comment (nvkelso) [inline docs/layers.md]: This list should be alphabetically sorted.

**Pull request merged**